### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by eritikass**
https://trello.com/c/dfou752Y/24-add-prettier

### CODE REVIEW

**Review by JaanaTPT — COMMENTED** [review]
missing CI

**Review by Mary2021 — APPROVED** [review]

**Review by gregorkookmaa — APPROVED** [review]

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Run pretttier
```
Comment (JaanaTPT) [inline .github/workflows/prettier.yml:1]: typo

Comment (gregorkookmaa) [inline .github/workflows/prettier.yml:1]: Typo. Should be "Run prettier#

Comment (eritikass) [reply .github/workflows/prettier.yml]: tnx fixed -> 8c7f841

Comment (eritikass) [reply .github/workflows/prettier.yml]: tnx fixed -> 8c7f841

**Review by Mary2021 — APPROVED** [review]

**Review by JaanaTPT — APPROVED** [review]

**Review by kpuistaja — APPROVED** [review]

**Review by Onne164 — APPROVED** [review]

**Review by AndreKangas — APPROVED** [review]

**Review by Ants-Kattel — APPROVED** [review]

**Pull request merged**